src/paste.rs

```diff
@@ -214,8 +213,6 @@ impl Paste {
             .await
             .map_err(error::ErrorInternalServerError)?;
         let payload_limit = config
-            .read()
-            .map_err(|_| error::ErrorInternalServerError("cannot acquire config"))?
```

Comment:
Why did you remove the error message? I can see that you removed all of these error messages, even tho8ugh this specific message is not moved to somewhere else.

Reply:
Because `tokio::sync::RWLock::read` doesn't have an error state which needs handling

Reply:
what if the file is missing or the permissions are wrong?

Reply:
ok, this makes sense